public/app/features/alerting/unified/components/rules/RulesGroup.tsx

```diff
@@ -289,11 +317,15 @@ export const RulesGroup = React.memo(({ group, namespace, expandAll, viewMode }:
       )}
       {isReorderingGroup && dsFeatures?.rulerConfig && (
         <ReorderCloudGroupModal
-          group={group}
+          group={{
+            name: group.name,
+            interval: group.interval,
+            rules: group.rules.map((r) => r.rulerRule).filter((rule) => rule !== undefined),
```

Comment:
nit: can we add a comment here to explain why this `rule` might be undefined?

Reply:
I believe that with the new group pages this comment is no longer relevant 🙂